test/language/chinese-reading-normalizer.test.js

```diff
@@ -25,6 +25,9 @@ const tests = [
     ['wán:zhěng', 'wánzhěng'],
     ['fān・yì', 'fānyì'],
     ['fān//yì', 'fānyì'],
+    ['fān’yì', 'fānyì'],
+    ['fān'yì', 'fānyì'],
```

Comment:
You need to escape the `'` here, otherwise the string gets broken. That's why CI is failing